mpi/outer_script.sh

```diff
@@ -35,7 +35,7 @@ COMPILERS="${LCSCHEDCLUSTER}_COMPILERS"
 for mpi in ${!MPIS}; do
     for compiler in ${!COMPILERS}; do
         if [[ $mpi == "cray-mpich" ]]; then
-            EXTRA_FLUX_SUBMIT_OPTIONS="-o pmi=cray-pals" flux batch -N2 -n4 --flags=waitable --output=kvs $MPI_TESTS_DIRECTORY/inner_script.sh $compiler $mpi
+            EXTRA_FLUX_SUBMIT_OPTIONS="-o pmi=cray-pals" flux batch --exclusive -N2 --flags=waitable --output=kvs $MPI_TESTS_DIRECTORY/inner_script.sh $compiler $mpi
```

Comment:
Do you want to drop the `-n4`?

Reply:
When I had both `--exclusive` and `-n4` I was getting full (n=48) node behavior anyway. I had `-n4` after `--exclusive`, would flipping the ordering give me different behavior?

Reply:
There should be no difference between `--exclusive -n4` and `-n4 --exclusive`. I don't think the argument parsing library checks order for optional flags like that.

I forgot that this was `flux batch` used here rather than `flux run` or `submit` so yeah it makes sense to drop the `-n4`.